.github/workflows/test.yaml

```diff
@@ -139,8 +191,15 @@ jobs:
       run: |
         ${{ matrix.tox-wrapper }} tox ${{ matrix.trial-target }}
 
+    # If one of the above steps fails, fire up tmate for remote debugging.
+    - name: Tmate debug on failure
+      if: ${{ !cancelled() && env.TMATE_DEBUG == 'yes' }}
+      uses: mxschmitt/action-tmate@v3
+      with:
+        limit-access-to-actor: true
+
     - name: Prepare coverage
-      if: ${{ !cancelled() && contains(matrix['tox-env'], 'withcov') }}
+      if: ${{ !cancelled() && !matrix.skip-coverage }}
```

Comment:
I very much like this change to use a structured value (a boolean input parameter to the job) instead of some string scraping to determine the job configuration.  Thank you for this.